code/modules/vehicles/cars/clowncar.dm

```diff
@@ -49,6 +49,24 @@
 /obj/vehicle/sealed/car/clowncar/mob_forced_enter(mob/M, silent = FALSE)
 	. = ..()
 	playsound(src, pick('sound/vehicles/clowncar_load1.ogg', 'sound/vehicles/clowncar_load2.ogg'), 75)
+	if(iscarbon(M))
+		var/mob/living/carbon/forced_mob = M
+		if(forced_mob.has_reagent(/datum/reagent/consumable/ethanol/irishcarbomb))
+			var/reagent_amount = forced_mob.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/irishcarbomb)
+			forced_mob.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
```

Comment:
Why bother deleting the reagent?

Reply:
If you get knocked out of a clown car and picked up again, it will trigger twice otherwise.

Reply:
Sounds fair enough, make sure this is only done for their mob reagents though instead of both them and the stomach.